packages/repository/src/model.ts

```diff
@@ -273,7 +273,7 @@ export abstract class ValueObject extends Model implements Persistable {}
 /**
  * Base class for entities which have unique ids
  */
-export abstract class Entity extends Model implements Persistable {
+export class Entity extends Model implements Persistable {
```

Comment:
Can you have a separate PR for this? Is it still required with `BaseModel`?

Reply:
> Can you have a separate PR for this? Is it still required with `BaseModel`?

I just snuck the commit in there so I can run the CI. I ran the tests locally but they keep dying on http proxy and example.com .  This change is not currently part of my proposed solution. I use BaseModel extends Entity {} right now.

Reply:
If so, please revert the change.

Reply:
Reverted the change. No longer exploring the idea of making `Entity` non-abstract.